apps/docs/content/guides/platform/org-mfa-enforcement.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: 'Enforce MFA on Organization'
+description: 'All users in an organization must have a valid MFA session to interact with organization resources'
+---
+
+Supabase provides multi-factor authentication (MFA) enforcement on the organization level. Ensuring all organization members must use MFA and have a valid MFA backed session when interacting with the organization and projects belonging to the organization.
+
+<Admonition type="note">
+
+MFA enforcement is only available on the [Pro, Team and Enterprise plans](https://supabase.com/pricing).
+
+</Admonition>
+
+When you first create an account, a default organization is created for you and you'll be assigned as the **Owner**. Any organizations you create will assign you as **Owner** as well.
```

Comment:
```suggestion
```

Any particular reason this is here? Seems like a bit of a non-sequitur

Reply:
🤦🏼‍♂️ 😅  copy-paste giving me away